### PR TITLE
allow LayeredModel to run with default arguments

### PR DESCRIPTION
**Pull request by asross**
(Based on <https://pyqg.readthedocs.io/en/latest/examples/layered.html> -- adding these defaults makes it much easier to write tests that invoke LayeredModel without adding a lot of boilerplate.)

### CODE REVIEW

File: pyqg/layered_model.py [hunk 1]
```diff
@@ -113,15 +114,24 @@ def __init__(
         self.beta = beta
         self.rd = rd
         self.delta = delta
+
+        if U is None: U = (np.arange(nz) * 0.025)[::-1]
+        if V is None: V = np.zeros(nz)
+        if H is None: H = [500] + [1750 for _ in range(nz-1)]
+        if rho is None: rho = np.arange(nz) * 0.3 + 1025
```
Comment (rabernat) [inline pyqg/layered_model.py:121]: Why not just use default arguments in the constructor?

**Review by rabernat — COMMENTED** [review]
This is a good idea Andrew. Just one observation...

File: pyqg/layered_model.py (hunk 1 above)
Comment (asross) [reply pyqg/layered_model.py:121]: Hmm -- that would be better in some respects, but using `None` and overwriting in the body of `__init__` allows `LayeredModel` to work no matter what `nz` is passed.

Comment (rabernat) [reply pyqg/layered_model.py:119]: gotcha that makes total sense.

Comment (asross) [reply pyqg/layered_model.py:119]: Awesome, will plan to merge then.

**Pull request merged**